Gothic-Unity-Project/Gothic-Unity
# Feature/177 quest log v2 + Hero Status menu v1

## To test
* Open game
* Grab on your left and right chest
* Will you hold the appropriate menus in your hands?
* Fine-tuning
    * Can you grab the menus from your body where you expect it?
    * Is the grab spot too wide or narrow?
    * Is your hand snapping at a spot where you'd expect it to be?
    * No blocking collision with your body when you pull the menu in your face?


Hints:
* In our next branch, we will remove the (unfinished) _grab from chest_ and put the menus open when main menu pops up.
